content/en/platform/corda/4.12/community/node-upgrade-notes.md

```diff
@@ -63,9 +64,9 @@ We provide some [backup recommendations]({{< relref "node-administration.md#back
 
 Download the required version of Corda Community Edition; for more information, see [Release packs and checksums]({{< relref "release-checksum-os.md" >}}).
 
-## Step 4: Upgrade the node to Corda 4.0 or later
+## Step 4: Upgrade the node to Corda 4.12 or later
```

Comment:
Think this should be 4.11. That is, i think the intention of this step is to place the node at a version where the new version can be upgraded., so that's why I think it's suppose to be 4.11.

Also not that line 26 also needs to be updated to match this section heading.